test-cases/Features/Users/Register/TC-E2E-009-boundary-max-length.md

```
@@ -0,0 +1,45 @@
+## Steps
+
+### BLOCK: First Name
+1. Generate 100-character string
+2. Enter in First Name
+3. Submit form
+4. Verify no error shown
+5. Repeat with 101 characters
+6. Verify validation error shown
+
+### BLOCK: Last Name
+7. Generate 100-character string
```

markdownlint-cli2 (0.20.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 7; Style: 1/2/3

(MD029, ol-prefix)